sdk/attestation/azure-security-attestation/test/samples/attestation/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.13)
+
+project (attestation-basic-operations LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+add_executable (
+  attestation-attestation
```

Comment:
XD, this is funny :).   Would it make sense to use some synonym? or even `attestation-hello-world` ?

Reply:
no convention.  It's just curious how the name is duplicated for the service name and the sample name.  So I was wondering if it would be worth it to use a different name for the sample name, like `attestation-bigbang`  :P.

Reply:
I don't know what the `bigbang` function is. The APIs which are being called are the attestation APIs.

Having said that, I think the convention for samples is going to align with one sample per scenario and one executable per sample. So this will get refactored to attestation-attestSgx, attestation-attestSgxWithRuntimeBinary, attestation-attestOpenEnclaveWithRuntimeJson, etc. 

There will probably end up being about 20 or so sample executables before I'm done, but that's ok.

I'll deal with fixing that up next week - I have a few days booked to flesh out the samples.